apps/hosts/admin.py

```diff
@@ -1,3 +1,19 @@
 from django.contrib import admin
 
-# Register your models here.
+from base.admin import TimeStampedAdmin
+
+from .models import ChallengeHostTeam, ChallengeHost
+
+
+@admin.register(ChallengeHostTeam)
+class HostTeamModelAdmin(TimeStampedAdmin):
```

Comment:
class name should be ChallengeHostTeamAdmin. convention ideally is name of model + 'Admin'